src/eventSourcing/Event.js

```diff
@@ -46,10 +52,17 @@ const DATA_SCHEMAS = {
   [TYPES.GATEWAY_ACTIVATED]: Joi.object({
     gatewayId: Joi.string().required(),
   }),
+  [TYPES.DISPATCHED_EVENT]: Joi.object({
+    eventDispatcherId: Joi.string().required(),
+  }),
 }
 
 const SCHEMA = BASE_SCHEMA.concat(Joi.object({
```

Comment:
This schema should be 100% usable as the model schema for the planned `ProcessEvent` model in hambda.